packages/devtools_app/lib/src/charts/chart.dart

```diff
@@ -7,11 +7,26 @@ import 'dart:ui';
 
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
+import 'package:intl/intl.dart' as intl;
 
 import '../auto_dispose_mixin.dart';
 import 'chart_controller.dart';
 import 'chart_trace.dart';
 
+/// Helper function to:
```

Comment:
nit: make this a proper doc comment. First line should summarize what the method does.
E.g.
`/// Perform some draw operations on a canvas after applying translate.`

Reply:
Done.